src/components/Challenge/index.js

```diff
@@ -0,0 +1,35 @@
+import {useNavigate} from 'react-router-dom';
+
+import {useStore} from '../../hooks/useStore';
+import Button from '../Button/index';
+import {ChallengeContainer} from '../Challenge/styled';
+
+export default function Challenge({
+	challengeId,
+	image,
+	title,
+	startingPoint,
+	endingPoint,
+	distance,
+}) {
+	const navigate = useNavigate();
+	const setActivChallengeId = useStore(state => state.setActivChallengeId);
+	return (
+		<ChallengeContainer>
+			<img src={image} aria-label={title} alt={title} />
+			<h2>{title}</h2>
+			<p>
+				{startingPoint} to {endingPoint}
+			</p>
+			<p>{distance} km</p>
+			<Button
+				onClick={() => {
+					setActivChallengeId(challengeId);
+					navigate('/ActiveChallenge');
+				}}
+			>
```

Comment:
I think it would be better to use a `Link` here instead of a `button`. I suggest you to work with `useParams` from React Router to add the id of the active challenge to the URL. In this way you also will not need the `acticeChallenge` state anymore.